translation/source/mobile.xml

```diff
@@ -5,4 +5,58 @@
   <string name="toolsTab" comment="The tab in bottom navigation bar. Must be very short (10 characters max) to avoid ugly text shifting issues." maxLength="10">Tools</string>
   <string name="watchTab" comment="The tab in bottom navigation bar. Must be very short (10 characters max) to avoid ugly text shifting issues." maxLength="10">Watch</string>
   <string name="settingsTab" comment="The tab in bottom navigation bar. Must be very short (10 characters max) to avoid ugly text shifting issues." maxLength="10">Settings</string>
+  <string name="mustBeLoggedIn" comment="This is shown when the user tries to access a feature that requires login.">You must be logged in to view this page.</string>
+  <string name="systemColors" comment="The title of the system colors setting. When enabling this, tha app's colors will match those of the operating system.">System colors</string>
+  <string name="feedbackButton" comment="Shown in the settings, pressing this button will redirect to the lichess contact site.">Feedback</string>
+  <string name="okButton" comment="This is used in varius confirmations dialogs. Translation should be short.">OK</string>
+  <string name="settingsHapticFeedback" comment="Title used in the settings. When enabled, the device will vibrate when making a move.">Haptic feedback</string>
+  <string name="settingsImmersiveMode" comment="Title in the settings. When enabled, the app will hide the navigation bar and status bar.">Immersive mode</string>
+  <string name="settingsImmersiveModeSubtitle" comment="Subtitle for the 'Immersive Mode' setting. Gives an explanation of what the button does.">
+      Hide system UI while playing. Use this if you are bothered by the system's navigation gestures at the edges of the screen. Applies to game and Puzzle Storm screens. 
+  </string>
+  <string name="notFollowingAnyUser" comment="This is shown in the list that contains user followed by the user. Displayed if the user does not follow any users.">You are not following any user.</string>
+  <string name="couldNotLoadRecentGames" comment="This is shown when the app fails to load the user's recent games.">Could not load recent games.</string>
```

Comment:
I'd say let's remove all strings of type "Could not load X". 

The idea is to not translate errors messages that are not critically important. Translating work is done by a team of volunteers and we should try to focus on quality rather than quantity. Such messages are mostly edge cases in the sense they should be rarely seen and people would understand that sth went wrong and could not load without understanding the actual meaning of the error message.

Reply:
This should be part of the internationalization documentation, which I have yet to write. Sorry for the unnecessary work.

Reply:
There's a retry screen with a generic error that should be translated:

https://github.com/lichess-org/mobile/blob/main/lib/src/widgets/feedback.dart#L107

The current message is 'Something went wrong', maybe we can write a slightly better one? the idea is that it must stay generic so it can be used in a lot of places.

Not a priority, but we could eventually replace those error widgets that just contain the text "Could not load X" by a generic retry widget in the same spirit as the full screen retry.

Reply:
No problem! I added a modified version of your comment to the docs now